distribution/lib/Standard/Image/0.0.0-dev/src/Image_File_Format.enso

```diff
@@ -0,0 +1,29 @@
+from Standard.Base import all
+import Standard.Base.Error.Illegal_Argument.Illegal_Argument
+
+import project.Data.Image.Image
+
+supported = [".bmp", ".dib", ".jpeg", ".jpg", ".jpe", ".jp2", ".png", ".webp", ".pbm", ".pgm", ".ppm", ".pxm", ".pnm", ".pfm", ".sr", ".ras", ".tiff", ".tif", ".exr", ".hdr", ".pic"]
```

Comment:
Where does that list come from? May be worth adding a comment from where it originates in case we need to debug sth.

Reply:
Inside the source code of OpenCV.

Reply:
Then I'd appreciate adding a comment here stating a path allowing to find this within OpenCV sources. Will save anyone debugging it from doing the search that you did already.